tests/generators/bls/main.py

```diff
@@ -72,49 +78,44 @@ def hash_message(msg: bytes,
             int_to_hex(fq2.coeffs[0], F2Q_COEFF_LEN),
             int_to_hex(fq2.coeffs[1], F2Q_COEFF_LEN),
         ]
-        for fq2 in bls.utils.hash_to_G2(msg, domain)
+        for fq2 in bls.hash_to_curve.hash_to_G2(msg, DST)
     ]
 
 
-def hash_message_compressed(msg: bytes, domain: bytes) -> Tuple[str, str]:
+def hash_message_compressed(msg: bytes) -> Tuple[str, str]:
     """
     Hash message
     Input:
         - Message as bytes32
-        - domain as bytes8
     Output:
         - Message hash as a compressed G2 point
     """
-    z1, z2 = bls.utils.compress_G2(bls.utils.hash_to_G2(msg, domain))
+    z1, z2 = bls.point_compression.compress_G2(bls.hash_to_curve.hash_to_G2(msg, DST))
     return [int_to_hex(z1, G2_COMPRESSED_Z_LEN), int_to_hex(z2, G2_COMPRESSED_Z_LEN)]
 
 
 def case01_message_hash_G2_uncompressed():
     for msg in MESSAGES:
-        for domain in DOMAINS:
-            yield f'uncom_g2_hash_{encode_hex(msg)}_{encode_hex(domain)}', {
-                'input': {
-                    'message': encode_hex(msg),
-                    'domain': encode_hex(domain),
-                },
-                'output': hash_message(msg, domain)
-            }
+        yield f'uncom_g2_hash_{encode_hex(msg)}', {
+            'input': {
+                'message': encode_hex(msg),
+            },
+            'output': hash_message(msg)
+        }
 
 
 def case02_message_hash_G2_compressed():
     for msg in MESSAGES:
-        for domain in DOMAINS:
-            yield f'com_g2_hash_{encode_hex(msg)}_{encode_hex(domain)}', {
-                'input': {
-                    'message': encode_hex(msg),
-                    'domain': encode_hex(domain),
-                },
-                'output': hash_message_compressed(msg, domain)
-            }
+        yield f'com_g2_hash_{encode_hex(msg)}', {
+            'input': {
+                'message': encode_hex(msg),
+            },
+            'output': hash_message_compressed(msg)
+        }
 
 
 def case03_private_to_public_key():
-    pubkeys = [bls.privtopub(privkey) for privkey in PRIVKEYS]
+    pubkeys = [bls.G2Basic.PrivToPub(privkey) for privkey in PRIVKEYS]
```

Comment:
```suggestion
    pubkeys = [bls. G2ProofOfPossession.PrivToPub(privkey) for privkey in PRIVKEYS]
```

Not that it matters from a functionality perspective, but I recon it should be `G2ProofOfPossession` everywhere as that is the specific ciphersuite implemented.